deployments/azure-webapps-dotnet-core.yml

```diff
@@ -0,0 +1,76 @@
+# This workflow will build and push a .NET Core app to an Azure Web App when a commit is pushed to your default branch.
+#
+# This workflow assumes you have already created the target Azure App Service web app.
+# For instructions see https://docs.microsoft.com/en-us/azure/app-service/quickstart-nodejs?tabs=linux&pivots=development-environment-cli
```

Comment:
🚗 Another drive by comment as I refer to this PR for the docs: You probably want to update the link here and elsewhere to the relevant guide.

Reply:
Good catch, thanks! Just fixed those links